### PR TITLE
triangle 1.0.3 (new formula)

### PR DESCRIPTION
**Pull request by equal-l2**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by equal-l2** [comment]
This software requires an image file for testing.
Does anyone have an idea from where I get such an image?

**Comment by lembacon** [comment]
> This software requires an image file for testing.
> Does anyone have an idea from where I get such an image?

Homebrew does provide a few image files for testing purposes where you can find here https://github.com/Homebrew/brew/tree/master/Library/Homebrew/test/support/fixtures.

You may access them via something like `test_fixtures("test.jpg")` or `test_fixtures("test.png")`.

**Comment by equal-l2** [comment]
>Homebrew does provide a few image files for testing purposes

Good to know, thanks!

File: Formula/triangle.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Triangle < Formula
+  desc "Convert images to computer generated art using delaunay triangulation"
```
Comment (fxcoudert) [inline Formula/triangle.rb]: `Delaunay` (https://en.wikipedia.org/wiki/Boris_Delaunay)

File: Formula/triangle.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = bin
+
+    dir = buildpath/"src/github.com/esimov/triangle"
+
+    dir.install buildpath.children
+
+    cd dir/"cmd/triangle" do
+      system "go", "install"
```
Comment (fxcoudert) [inline Formula/triangle.rb]: Does this include or download any resources?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/triangle.rb (hunk 2 above)
Comment (equal-l2) [reply Formula/triangle.rb]: Probably not. The archive contains `vendor` directory for dependencies.
(I can successfully install the formula without an internet connection.)

File: Formula/triangle.rb (hunk 1 above)
Comment (equal-l2) [reply Formula/triangle.rb]: Fixed.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @equal-l2 for the new formula, merging!

**Pull request closed without merging**